src/webapp/reports/validate-custom-forms/ValidateCustomFormsReport.tsx

```diff
@@ -0,0 +1,99 @@
+import { Typography, makeStyles } from "@material-ui/core";
+import _ from "lodash";
+import React, { useState } from "react";
+import i18n from "../../../locales";
+import { Spinner } from "../../components/objects-list/Spinner";
+import { Select, SelectOption } from "../../components/select/Select";
+import { useAppContext } from "../../contexts/app-context";
+
+const list = [{ text: "" }];
```

Comment:
This `list` is being modified inplace in the component, that cannot be done. The react way is to use a state: `const [list, setList] = React.useState<Array<{text: string}>>([])`. Also, `list` is not a good name, it says the shape of the object (that we already know because it's typed), we should say what contains. Here: `errors`?